homepage/tests/followedUserCourses_model_tests.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from homepage.models import AppUser, Course, FollowedUserCourses
+
+
+@pytest.mark.django_db
+class TestFollowedUserCourses:
+
+    # -----------------------------------------------#
+    # create the test data (appUser, courses, user_course_pairs) that will be used in the test methods
+    @pytest.fixture
+    def appUsers(self):
+        appUser0 = AppUser.create_appUser('user0', 'user1@mta.ac.il', '1234')
+        appUser1 = AppUser.create_appUser('user1', 'user2@gmail.com', '1234')
+        appUser_notFollowing = AppUser.create_appUser('user_notFollowing', 'user3@', '9722')
+
+        return (appUser0, appUser1, appUser_notFollowing)
+
+    @pytest.fixture
+    def courses(self):
+        course0 = Course(course_id=10231, name='course0', mandatory=True, credit_points=0, avg_load=0,
+                         avg_rating=0, num_of_raters=0, num_of_reviewers=0)
+        course0.save()
+
+        course1 = Course(course_id=10400, name='course1', mandatory=True, credit_points=0,
+                         avg_load=0, avg_rating=0, num_of_raters=0, num_of_reviewers=0)
+        course1.save()

```

Comment:
```suggestion
        course1 = Course(course_id=10400, name='course1', mandatory=True, credit_points=0,
                         avg_load=0, avg_rating=0, num_of_raters=0, num_of_reviewers=0).save()

```

Reply:
This doesn't work here (at least for me).
maybe .save() does not return a 'Course' object?

![image](https://user-images.githubusercontent.com/79373827/117184034-f0d98980-ade0-11eb-9628-451ff946530f.png)

Reply:
Update: i think i know why its not possible to do .save() on the same line here-
https://docs.djangoproject.com/en/3.2/ref/models/instances/#django.db.models.Model.save

![image](https://user-images.githubusercontent.com/79373827/117186502-9261da80-ade3-11eb-8361-2002a1bc407d.png)